tests/gateway/test_passthrough_proxy.py

```
@@ -0,0 +1,360 @@
+        if server._session:
+            await server._session.close()
+        if server._runner:
+            await server._runner.cleanup()
+
+    async def test_passthrough_forward(self, running_proxy):
+        """Request forwarded correctly to upstream.
+
+        Verifies that requests are forwarded to the upstream API
+        and responses are returned unchanged.
+        """
+        server, base_url = running_proxy
```

Ruff (0.14.10)

Unpacked variable `server` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)